Packs/Jira/Scripts/JiraCreateIssueExample/JiraCreateIssueExample_test.py

```diff
@@ -1,82 +1,134 @@
+    (["customfield_10096=test", "customfield_10040=A100"], [{"customfield_10096": "test"}, {"customfield_10040": "A100"}]),
+    (["customfield_10096:test", "customfield_10040=A100"], [{"customfield_10040": "A100"}]),
+    ([], []),
 ])
-def test_get_additional_args_from_input(input, expected):
+def test_parse_custom_fields(custom_fields, expected):
+
     """
     Given:
-        - A dictionary view with Jira fields
+        - A list of strings of custom fields.
+        - An expected list of dicts of custom fields.
+
     When:
-        - 1 unknown argument key is supplied
-        - 1 unknown argument key is supplied, 1 known argument key is supplied
-        - 1 known arguments are supplied, 0 unknown argument key is supplied
+        - Case A: 1 text custom field.
+        - Case B: 1 text custom field, 1 integer custom field.
+        - Case C: 1 text custom field, 1 integer custom field with 0 padding.
+        - Case D: 2 text custom fields.
+        - Case E: 1 text custom field, 1 custom field with unexpected delimiter (:).
+        - Case F: Empty custom field list.
 
     Then:
-        - empty dict
-        - dict with unknown argument key is returned
-        - dict with unknown argument key is returned
+        - Case A: 1 text custom field returned.
+        - Case B: 1 text custom field, 1 integer custom field returned.
+        - Case C: 1 text custom field, 1 integer custom field returned.
+        - Case D: 2 text custom fields returned.
+        - Case E: 1 text custom field returned.
+        - Case F: Empty custom field list returned.
     """
 
-    args = get_additional_args_from_input(input)
+    actual = parse_custom_fields(custom_fields)
 
-    assert isinstance(args, dict)
-    assert args == expected
+    assert len(actual) == len(expected)
+    assert actual == expected
 
 
-@pytest.mark.parametrize("known,additional,expected", [
-    ({DEFAULT_ARGS[0]: "system down"}, None, {DEFAULT_ARGS[0]: "system down"}),
-    ({DEFAULT_ARGS[0]: "system down"}, {"unknown_key": "some key"},
-        {'issueJSON': '{"unknown_key": "some key"}', DEFAULT_ARGS[0]: "system down"}),
+@pytest.mark.parametrize("args, custom_fields, expected", [
+    ({"arg1": "val1", "arg2": 1}, [{"customfield_10096": "test"}, {"customfield_10040": 100}],
+        {"arg1": "val1", "arg2": 1, "issueJson": {"fields": {"customfield_10096": "test", "customfield_10040": 100}}}),
+    ({"arg1": "val1", "arg2": 1}, [],
+        {"arg1": "val1", "arg2": 1}),
+    ({}, [{"customfield_10096": "test"}, {"customfield_10040": 100}],
+        {"issueJson": {"fields": {"customfield_10096": "test", "customfield_10040": 100}}})
 ])
-def test_add_additional_args(known, additional, expected):
+def test_add_custom_fields(args, custom_fields, expected):
     """
     Given:
-        - 2 dictionaries of known and unknown keys
-    When:
-        - 1 known dict, no unknown dicts
-        - 1 known dict and one unknown dict
+        - A dictionary of arguments.
```

Comment:
Same comments as above

Reply:
Addressed in [`2b875a6` (#22601)](https://github.com/demisto/content/pull/22601/commits/2b875a6111d34fc492e18ac7ee5335e81c46a278).